src/shared/meshes/cell_linked_list.h

```diff
@@ -112,6 +155,16 @@ class CellLinkedList : public BaseCellLinkedList, public Mesh
     void InsertListDataEntry(size_t particle_index, const Vecd &particle_position, Real volumetric) override;
     virtual ListData findNearestListDataEntry(const Vecd &position) override;
     virtual StdLargeVec<size_t> &computingSequence(BaseParticles &base_particles) override;
+    template<class ExecutionPolicy>
+    StdLargeVec<size_t> &computingSequence(BaseParticles &base_particles, ExecutionPolicy)
+    {
+        /*StdLargeVec<Vecd> &pos = base_particles.pos_;
+        StdLargeVec<size_t> &sequence = base_particles.sequence_;
+        size_t total_real_particles = base_particles.total_real_particles_;
+        particle_for(execution_policy, total_real_particles, [&](size_t i, auto&& kernel)
+            { sequence[i] = transferMeshIndexToMortonOrder(CellIndexFromPosition(pos[i])); }, ExecutionSelector<BaseMesh, ExecutionPolicy>(this));
+        return sequence;*/
+    }
```

Comment:
Seems that all the contents in this functions is commented.

Reply:
Yes, I should have kept it for the next PR instead of commenting it.
Essentially, I am now working on particle sorting on sycl, and since sorting is also connected to the cell linked list I started working on both at the same time, so some code regarding the next PR slipped into this one.